rag/graphrag/general/index.py

```
@@ -123,28 +125,27 @@ def _select_extractor(graphrag_config: dict):
     """Return the extractor class matching ``graphrag_config["method"]``.
 
     Supported values:
-    - ``"general"``  – Microsoft GraphRAG LLM-based extractor (default in
-      earlier versions).
-    - ``"light"``   – LightRAG-style LLM-based extractor (the default when
-      *method* is omitted or unrecognised).
-    - ``"ner"``     – NER-based extractor using spaCy (no LLM
-      needed for entity / relation extraction itself).
+    - ``"general"``  – Microsoft GraphRAG LLM-based extractor.
```

Ruff (0.15.21)

[warning] Docstring contains ambiguous `–` (EN DASH). Did you mean `-` (HYPHEN-MINUS)?

(RUF002)